babelon/hp-it.babelon.tsv

```diff
@@ -1,2900 +1,3489 @@
 source_language	source_value	subject_id	predicate_id	translation_language	translation_value	translation_status	translator	translator_expertise	comment	translation_date
-en	Hydrocephalus	HP:0000238	rdfs:label	it	Idrocefalo	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Severe global developmental delay	HP:0011344	rdfs:label	it	Grave ritardo globale dello sviluppo	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Retinal hemorrhage	HP:0000573	rdfs:label	it	Emorragia retinica	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Intracranial hemorrhage	HP:0002170	rdfs:label	it	Emorragia intracranica	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Delayed speech and language development	HP:0000750	rdfs:label	it	Ritardo nello sviluppo del linguaggio e della parola	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Bulbous nose	HP:0000414	rdfs:label	it	Naso bulboso	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Hypotonia	HP:0001252	rdfs:label	it	Ipotonia	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
+en	Multicystic kidney dysplasia	HP:0000003	rdfs:label	it	Displasia renale multicistica	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
+en	Recurrent urinary tract infections	HP:0000010	rdfs:label	it	Infezioni ricorrenti del tratto urinario	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
+en	Neurogenic bladder	HP:0000011	rdfs:label	it	Vescica neurogena	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
+en	Urinary urgency	HP:0000012	rdfs:label	it	Urgenza urinaria	OFFICIAL	0000-0003-4174-0526	PHYSICIAN		2024-11-26
```

Comment:
Can we do `orcid:0000-0003-4174-0526` instead of `0000-0003-4174-0526` for sustainability?

Reply:
done in https://github.com/obophenotype/hpo-translations/pull/20/commits/4d224123c493799b6a0140656d04d9741b6853ad